I'm sorry, but an uncaught exception occurred.

Compiling ATL code at game/screen_snowflakes.rpy:12
  File "game/screen_snowflakes.rpy", line 37, in script
    "woho"
  File "renpy/common/000window.rpy", line 126, in _window_auto_callback
    _window_show(auto=True)
  File "renpy/common/000window.rpy", line 70, in _window_show
    renpy.with_statement(trans)
  File "game/screen_snowflakes.rpy", line 12, in <module>
    easein 5.0*(1.1-xya[1]) align ( xa + (renpy.random.random()-0.5)*0.3, 1.1)
NameError: name 'xa' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/screen_snowflakes.rpy", line 37, in script
    "woho"
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\ast.py", line 2560, in execute
    Say.execute(self)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\ast.py", line 585, in execute
    statement_name("say")
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\ast.py", line 52, in statement_name
    i(name)
  File "renpy/common/000window.rpy", line 126, in _window_auto_callback
    _window_show(auto=True)
  File "renpy/common/000window.rpy", line 70, in _window_show
    renpy.with_statement(trans)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\exports.py", line 1782, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\core.py", line 1549, in do_with
    return self.interact(trans_pause=True,
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\core.py", line 2166, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\core.py", line 2688, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\screen.py", line 485, in visit_all
    self.child.visit_all(callback, seen=None)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\display\displayable.py", line 421, in visit_all
    for d in self.visit():
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\atl.py", line 764, in visit
    block = self.compile()
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\atl.py", line 671, in compile
    block = self.atl.compile(self.context)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\atl.py", line 873, in compile
    statements = [ i.compile(ctx) for i in self.statements ]
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\atl.py", line 873, in <listcomp>
    statements = [ i.compile(ctx) for i in self.statements ]
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\atl.py", line 1214, in compile
    value = ctx.eval(expr)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\atl.py", line 392, in eval
    return renpy.python.py_eval(expr, locals=self.context)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\2022portable_programs\renpy-8.2.3-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screen_snowflakes.rpy", line 12, in <module>
    easein 5.0*(1.1-xya[1]) align ( xa + (renpy.random.random()-0.5)*0.3, 1.1)
NameError: name 'xa' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.3.24061702
freezing_love_1 1.0
Fri Dec 20 04:19:15 2024
